CMakeLists.txt

```diff
@@ -66,6 +66,9 @@ OPTION(WANT_WINMM	"Include WinMM MIDI support" OFF)
 OPTION(WANT_QT5		"Build with Qt5" OFF)
 
 
+SET(WINE_CXX_FLAGS "" CACHE STRING "Extra flags passed to wineg++")
```

Comment:
@jasp00 wouldn't this make more functional sense in the `IF(WANT_VST)` block, since we don't use Wine for anything else?